pghoard/restore.py

```diff
@@ -34,6 +34,11 @@ class RestoreError(Error):
     """Restore error"""
 
 
+def create_signal_file(file_path):
+    """Just ensure the file exists"""
+    open(file_path, "w")
```

Comment:
Missing close here. (I know CPython ends up closing this but that's not something that should be dependent on.)